autoscheduler/frontend/templates/index.html

```diff
@@ -4,8 +4,16 @@
         <meta charset="UTF-8" />
         <title> Auto Aggie Scheduler </title>
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+        <style>
```

Comment:
Rather than changing index.html, I could add global style rules to Schedule.css, but that would be harder to debug, and also, it's possible that we don't want any of our components to be larger than the window. In that case, this is better.